tests/functional_tests/autolab.sh

```diff
@@ -19,6 +19,13 @@ export TESTDIR
 # install node dependencies
 npm --quiet install 1>/dev/null 2>&1
 
```

Comment:
It is better to rename this file to `test.sh` in order to be consistent with the naming conventions of the orchestrating scripts for unit and deployment tests.